app/views/password/password_form.html.erb

```diff
@@ -4,20 +4,26 @@
 
 <h1 class="govuk-heading-l"><%= t 'password.change_password_heading' %></h1>
 
-<%= form_for @password_form, url: profile_change_password_path do |f| %>
+<%=error_summary_for(@password_form&.errors, :form) %>
+<%= form_for @password_form, url: profile_change_password_path, as: :form do |f| %>
   <fieldset class="govuk-fieldset" aria-describedby="changed-password-hint">
-    <div class="govuk-form-group">
-      <%= f.label :old_password, t('password.current_password_lbl'), class: "govuk-label" %>
-      <%= f.password_field :old_password, class: "govuk-input govuk-input--width-20" %>
+    <div class="govuk-form-group <%= 'govuk-form-group--error' if @password_form.errors.key?(:old_password) %>">
+      <h3 class="govuk-heading-m"><%= f.label :old_password, t('password.old_password_lbl'), class: "govuk-label" %></h3>
```

Comment:
these shouldn't be headings but labels, and be as per the design system to ensure accessibility and consistency